app/views/shared/my_facilities/_inactive_facilities.html.erb

```diff
@@ -19,11 +19,11 @@
       <% @inactive_facilities.sort_by(&:name).each do |facility| %>
         <tr>
           <td><%= link_to facility.name, analytics_facility_path(facility) %></td>
-          <td class="text-right"><%= @inactive_facilities_bp_counts[:last_week][facility.id] %></td>
-          <td class="text-right"><%= @inactive_facilities_bp_counts[:last_month][facility.id] %></td>
+          <td class="text-right"><%= @inactive_facilities_bp_counts[:last_week][facility.id].to_i %></td>
+          <td class="text-right"><%= @inactive_facilities_bp_counts[:last_month][facility.id].to_i %></td>
```

Comment:
Why did we add `.to_i` here? To ensure we show `0` if nil?

Reply:
Yes, the `to_i` makes the `nil` a zero.